website/config.toml

```diff
@@ -6,6 +6,7 @@ baseURL = "/"
 disableKinds = ["taxonomy", "taxonomyTerm"]
 
 # Language settings
+enableRobotsTXT = true
```

Comment:
Does this mean the robots.txt that's committed was generated by Hugo?

Reply:
no. it just means that the committed robots.txt makes it into the live site.